libraries/chain_kv/include/b1/session/session.hpp

```diff
@@ -139,7 +139,7 @@ class session {
 
    /// \brief Constructs a child session from another instance of the same session type.
    explicit session(session& parent, std::nullptr_t);
-   session(const session&) = delete;
+   //session(const session&) = delete;
```

Comment:
Do you know why you needed to remove this? I don't think you should need a copy constructor.

Reply:
Thanks! Tim also spot this. I have changed it back in the next commit.